packages/libs/react-components/package.json

```diff
@@ -57,6 +57,7 @@
     "@types/node": "^16.0.0",
     "@types/react": "^18.0.30",
     "eslint": "^8.15.0",
+    "eslint-import-resolver-alias": "~1.1.2",
```

Comment:
Why is this added? Can't we add it to the root config? Or will that not work with IDEs?

Reply:
it's unused, removed